harpoon/go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/cucumber/godog v0.14.1
 	github.com/cucumber/messages/go/v21 v21.0.1
 	github.com/olekukonko/tablewriter v0.0.5
-	github.com/redpanda-data/helm-charts v0.0.0-20241114193526-f53a0adc8f15
+	github.com/redpanda-data/helm-charts v0.0.0-20241121211850-232b4aaab5b2
```

Comment:
Should this be changed to 
```suggestion
	github.com/redpanda-data/redpanda-operator/pkg v0.0.0
```

And at the bottom have replace line to reference module.

Reply:
That will be an issue to fix later. One step at a time, this PR is just focused on getting `pkg` running.